scrapy/extensions/feedexport.py

```diff
@@ -241,55 +243,104 @@ def __init__(self, crawler):
 
         self.storages = self._load_components('FEED_STORAGES')
         self.exporters = self._load_components('FEED_EXPORTERS')
+        self.storage_batch_item_count = self.settings.get('FEED_STORAGE_BATCH_ITEM_COUNT', None)
```

Comment:
Would it be possible to have batch_item_count support in FEEDS setting?

Reply:
I guess so. Should I do it in this PR or it is already overloaded.

Reply:
It would be great if we could have it as part of this pull request.

Reply:
```suggestion
        self.storage_batch_item_count = self.settings.getint('FEED_STORAGE_BATCH_ITEM_COUNT')
```